src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+  const mediumTermTime = 3;
+  const mediumTermDiscount = 20;
+  const longTermTime = 7;
+  const longTermDiscount = 50;
+
+  if (days < mediumTermTime) {
+    return basePrice;
+  } else if (days >= mediumTermTime && days < longTermTime) {
+    return basePrice - mediumTermDiscount;
+  }
+
+  return basePrice - longTermDiscount;
```

Comment:
Similarly, the discount logic for long-term rentals should be adjusted to `longTermDiscount * days` to apply the discount for every day beyond the threshold.